phac-nml/irida-next
# Workflow Executions: Filter

## What does this PR do and why?
This PR adds filtering to the user and project's workflow execution index pages

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/e7193376-7487-485a-aeff-cabd8c748edf)

## How to set up and validate locally
For both the user and project's workflow execution index pages:
1. Create numerous workflow executions if necessary
2. Use the filter and ensure it filters as expected (by ID and name)
3. Ensure sort works in tandem with filter.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
